src/backend/file_vectorizer_main/file_vectorize_main.py

```diff
@@ -0,0 +1,19 @@
+import logging
+from src.backend.file_vectorizer.file_vectorizer import FileVectorizer
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+def main():
+    logger.info("Starting file vectorizer process...")
+    vectorizer = FileVectorizer()
+    while True:
+        try:
+            vectorizer.worker_loop()
+            logger.info("File vectorization completed successfully.")
+        except Exception as e:
```

Comment:
Lets set a delay of 2 seconds else the CPU will go high if there is continuous exception